index.js

```diff
@@ -0,0 +1,532 @@
+
+let capsColck = new btnRowThree ('CapsLock'),
+    a = new btnRowThree ('KeyA', 97, 1092),
+    s = new btnRowThree ('KeyS', 115, 1099),
+    d = new btnRowThree ('KeyD', 100, 1074),
+    f = new btnRowThree ('KeyF', 102, 1072),
+    g = new btnRowThree ('KeyG', 103, 1087),
+    h = new btnRowThree ('KeyH', 104, 1088),
+    j = new btnRowThree ('KeyJ', 106, 1086),
+    k = new btnRowThree ('KeyK', 107, 1083),
+    l = new btnRowThree ('KeyL', 108, 1076),
+    semicolon = new btnRowThree ('Semicolon', 59, 1078),
+    quote = new btnRowThree ('Quote', 39, 1101),
+    enter = new btnRowThree ('Enter');
+
+let shiftLeft = new btnRowFour ('ShiftLeft'),
+    z = new btnRowFour ('KeyZ', 122, 1103),
+    x = new btnRowFour ('KeyX', 120, 1095),
+    c = new btnRowFour ('KeyC', 99, 1089),
+    v = new btnRowFour ('KeyV', 118, 1084),
+    b = new btnRowFour ('KeyB', 98, 1080),
+    n = new btnRowFour ('KeyN', 110, 1090),
+    m = new btnRowFour ('KeyM', 109, 1100),
+    comma = new btnRowFour ('Comma', 44, 1073),
+    period  = new btnRowFour ('Period', 46, 1102),
+    slash  = new btnRowFour ('Slash', 47, 46),
+    arrowUp = new btnRowFour ('▲'),
+    shiftRight = new btnRowFour ('ShiftRight');
+
+let controlLeft  = new btnRowFive ('ControlLeft'),
+    win = new btnRowFive ('Win',),
+    altLeft  = new btnRowFive ('AltLeft'),
+    space = new btnRowFive ('Space'),
+    altRight = new btnRowFive ('AltRight'),
+    arrowLeft = new btnRowFive ('◀'),
+    arrowDown = new btnRowFive ('▼'),
+    arrowRight = new btnRowFive ('▶');
+    controlRight = new btnRowFive ('ControlRight');
+
+
+let pressShift = false;
+let pressCapsLock = false;
+let alterText = false;  
+let switchLanguage = false;
+
+root.prepend(textarea, keyboard);
+body.prepend(root);
+
+// reset event default fisical keybord
+document.onkeydown = (e) => e.code ? false : true;
+
+// switching style key
+const styleKeyDown = (e) => {
+    if(e.code == 'ShiftLeft' || e.code == 'ShiftRight' || e.code == 'CapsLock') return;
+    document.querySelector(`.keyboard .${e.code}`).classList.add('active');
+}
+
+// switching style key
+const styleKeyUp = (e) => {
+    if(e.code == 'ShiftLeft' || e.code == 'ShiftRight' || e.code == 'CapsLock') return;
+    document.querySelectorAll(`.active`).forEach(el => el.classList.remove('active'));
+}
+
+// switching style mouse
+const styleMouseDown = (e) => {
+    if(e.target.classList.contains('ShiftLeft') || e.target.classList.contains('ShiftRight')) return;
+    if(!e.target.classList.contains('btn') && !e.target.classList.contains('special--btn')) return;
+    e.target.classList.add('active');
+}
+
+// switching style mouse
+const styleMouseUp = (e) => {
+    if(e.target.classList.contains('ShiftLeft') || e.target.classList.contains('ShiftRight')) return;
+    document.querySelectorAll(`.active`).forEach(el => el.classList.remove('active'));
+}
+
+// add text in text area for key event;
+const innerTextAreaOnKeyDown = (e) => {
+
+    if(e.code != 'AltLeft') {
```

Comment:
the better way to delete a lot of if and start using object literals

https://ultimatecourses.com/blog/deprecating-the-switch-statement-for-object-literals